### DISCUSSION TITLE
review: psychiatric and mental disorders

### DISCUSSION THREAD

**Discussion by pnrobinson** [monarch-initiative/mondo#2678] [Mondo design decisions]
### psychosocial short stature

This should not be classified as a developmental disorder, even though logically it is that to a certain extent. There should be some subhierarchy of child abuse/neglect-related diseases. I did not see any of these terms at first glance

Here is a review article
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4288037/

**Comment by cmungall** [comment]
Would this proposed parent be a top level term, or under 
mental/psychiatric?

the whole of mental/psych diseases in mondo and all other 
ontologies/vocabs I have seen are not satisfactory, Nicole, can you 
create a gh project for these so we can organize tickets. Who can we 
bring in to help with this?

**Comment by nicolevasilevsky** [comment]
related to https://github.com/monarch-initiative/mondo/issues/130

**Comment by nicolevasilevsky** [comment]
From Peter:
The two classifications of mental disorder are debatable.

https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005084

It is not the case that every mental disorder is a psychiatric disorder, and this link is wrong.

Logically it is the case that every mental disorder is a brain disorder, but there are many cases of mental disorder where it is not possible to identify the "cause" in the brain, and so I am not sure this is the best primary classification. I do not have an easy answer right now, but this needs TLC.

(ticket https://github.com/monarch-initiative/mondo/issues/527)

**Comment by nicolevasilevsky** [comment]
Other related tickets:

- https://github.com/monarch-initiative/mondo/issues/130
- https://github.com/monarch-initiative/mondo/issues/2423

**Comment by nicolevasilevsky** [comment]
Should revise the psych hierarchy based on DSM-5
https://www.psychiatry.org/psychiatrists/practice/dsm

classify diseases based on chapter headings and the diseases that are mentioned in each chapter.

**Comment by nicolevasilevsky** [comment]
Need term
psychomotor disorder
https://my.clevelandclinic.org/health/diseases/21521-psychosomatic-disorder

**Comment by kanems** [comment]
UMLS lumps "psychiatric disorders" and "mental disorders" into one CUI (C0004936), which is in alignment with SNOMED CT 
74732009, MeSH D001523, NCI C2893... and possibly other sources in UMLS that I am less familiar with. 
I am not convinced this is correct, given the structure of Mondo's hierarchy under MONDO:0002025 psychiatric disorder, but I am not sure UMLS will split the CUI into two things when SNOMED CT lumps them. Just wanted to give you a heads up that MedGen will have to represent one of these concepts with a CN instead of a UMLS CUI. 
If you have any other sources/insight from the DSM-5 alignment work discussed here that we could share with UMLS, perhaps we could eventually get these terms clarified and harmonized.
